### PR TITLE
add new vendor specific event containing the packet error stats

### PR DESCRIPTION
**Pull request by hoarf**

### CODE REVIEW

File: hci/event/events/vendor_specific/hci_extension_packet_error_rate.py [hunk 1]
```diff
@@ -45,9 +45,9 @@ def num_misssed_events(self):
     def __str__(self):
         return super().__str__() + '\n' + '\n'.join([
             'Cmd OPCode: {} ({})',
```
Comment (pedrogyrao) [inline hci/event/events/vendor_specific/hci_extension_packet_error_rate.py]: "Command"

File: hci/event/events/vendor_specific/hci_extension_packet_error_rate.py [hunk 2]
```diff
@@ -45,9 +45,9 @@ def num_misssed_events(self):
...
-            'PacketErrorRate Command: {} ({})',
+            'Packet Error Rate Command: {} ({})',
             'Num Packets: {} ({})',
```
Comment (pedrogyrao) [inline hci/event/events/vendor_specific/hci_extension_packet_error_rate.py]: "Number of Packets"

File: hci/event/events/vendor_specific/hci_extension_packet_error_rate.py [hunk 3]
```diff
@@ -45,9 +45,9 @@ def num_misssed_events(self):
...
-            'Num Crc Error: {} ({})',
```
Comment (pedrogyrao) [inline hci/event/events/vendor_specific/hci_extension_packet_error_rate.py]: "Number of CRC Errors: "

File: hci/event/events/vendor_specific/hci_extension_packet_error_rate.py [hunk 4]
```diff
@@ -45,9 +45,9 @@ def num_misssed_events(self):
...
+            'Num CRC Error: {} ({})',
             'Num Events: {} ({})',
```
Comment (pedrogyrao) [inline hci/event/events/vendor_specific/hci_extension_packet_error_rate.py]: "Number of Events: "

File: hci/event/events/vendor_specific/hci_extension_packet_error_rate.py [hunk 5]
```diff
@@ -45,9 +45,9 @@ def num_misssed_events(self):
...
             'Num Missed Events: {} ({})']).format(
```
Comment (pedrogyrao) [inline hci/event/events/vendor_specific/hci_extension_packet_error_rate.py]: "Number of Missed Events"

**Review by pedrogyrao — CHANGES_REQUESTED** [review]

**Comment by hoarf** [comment]
requested changes made

**Review by pedrogyrao — APPROVED** [review]

**Pull request merged**